storages/memory-storage/Cargo.toml

```diff
@@ -1,26 +1,29 @@
 [package]
-name = "memory-storage"
-version = "0.0.0"
+name = "gluesql_memory_storage"
```

Comment:
It was named in crates.io with _ , so I couldn't change it.
rfc: <https://github.com/rust-lang/crates.io/issues/728>